www/.env.sample

```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH0_DOMAIN=
+REACT_APP_AUTH0_CLIENT_ID=
```

Comment:
It is secure to store `CLIENT_ID` in environment? This just gets baked into the app when shipped to the user right so technically it is public?

Reply:
I mean the client has to request a token from Auth0 anyway. I think it's safe to store it in the `.env` generated from Github secrets.

Reply:
Sounds good